pom.xml

```diff
@@ -15,7 +15,7 @@
 		<spring.version>3.2.11.RELEASE</spring.version>
 		<spring.security.version>3.2.5.RELEASE</spring.security.version>
 		<modelmapper.version>0.7.3</modelmapper.version>
-		<node.version>v6.9.1</node.version>
+		<node.version>v8.11.3</node.version>
```

Comment:
@IntegratedBreedingPlatform/developers 

After this is merged, if you have `node` installed globally please remember to upgrade the version too. I usually do it with `n`:
```shell
sudo n 8.11.3
```